shared_python/Tags.py

```diff
@@ -133,11 +132,13 @@ def update_tag_row(self, row: dict):
     ao3_tag_types = row[tag_headers['ao3_tag_type']].split(",")
     number_types = len(ao3_tag_types)
 
+    num_insert = 0
     # If tags length > types length -> there are remapped tags
     # Iterate over all the provided AO3 tags:
     # - First tag -> update the existing row
     # - Other tags -> create new row in tags table
     for idx, ao3_tag in enumerate(ao3_tags):
+      ao3_tag = ao3_tag.lstrip().rstrip()
```

Comment:
Looks as if Python's `strip` might do the trick here.

Reply:
yes it does! just replaced with `strip()`